src/prefect/orion/orchestration/dependencies.py

```diff
@@ -31,6 +33,28 @@ async def provide_flow_policy():
     return provided_policy
 
 
+async def provide_task_orchestration_parameters():
+    provided_parameters = ORCHESTRATION_DEPENDENCIES.get(
+        "task_orchestration_parameters"
+    )
+
+    if provided_parameters is None:
+        provided_parameters = dict()
+
+    return provided_parameters
```

Comment:
What do you think about having `ORCHESTRATION_DEPENDENCIES["task_orchestration_parameters"]` be a _function_ that returns the `dict` of parameters, instead of just a static `dict` of parameters?  Seems like it would cover more cases?

Reply:
absolutely!